httpapi/router.go

```diff
@@ -0,0 +1,36 @@
+package httpapi
+
+import (
+	"github.com/centrifuge/go-centrifuge/httpapi/health"
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+)
+
+// Router returns the http mux for the server.
+// @title Centrifuge OS Node API
+// @description Centrifuge OS Node API
+// @version 0.0.4
+// @contact.name Centrifuge
+// @contact.url https://github.com/centrifuge/go-centrifuge
+// @contact.email hello@centrifuge.io
+// @BasePath /
+// @license.name MIT
+// @host localhost:8082
+// @schemes http
+func Router(config Config) *chi.Mux {
+	r := chi.NewRouter()
+
+	// add middlewares
+	r.Use(middleware.Recoverer)
+	r.Use(middleware.DefaultLogger)
+
+	// health check
+	health.Register(r, config)
```

Comment:
Here we would centralize the registration of all services, right?

Reply:
yeap. Each one follows as we add more and this is locations where we update the swagger API version through comments. For now, we will fallback to the one we defined on the swagger.js

Reply:
That part is still not clear enough, how swagger generates the json out of comments, can you add an example here?

Reply:
This is through declarative comments: https://github.com/swaggo/swag#declarative-comments-format